mne/minimum_norm/spatial_resolution.py

```diff
@@ -146,30 +146,32 @@ def _localisation_error(resmat, src, function, metric):
     resmat = _rectify_resolution_matrix(resmat)
     locations = _get_src_locations(src)  # locs used in forw. and inv. operator
     locations = 100. * locations  # convert to cm (more common)
-    resmat = np.absolute(resmat)  # only use absolute values
+    # we want to use absolute values, but doing abs() mases a copy and this
+    # can be quite expensive in memory. So let's just use abs() in place below.
```

Comment:
FYI @olafhauk these matrices can be quite large (~8000x8000), so instead of doing this operation here, I've moved it to be done column-wise. It shouldn't be an appreciable performance difference but it should lower memory usage by ~800 MB in the `plot_resolution_metrics.py` example.